docs/load-balancer/sticky.md

```diff
@@ -0,0 +1,29 @@
+# Sticky Load Balancing
+
+The sticky load balancer selects a single service instance and keeps using it until it fails.
+Then, it selects the next one.
+
+It is possible to configure a back off time to specify for how long a service instance that failed should not be retried.
+
+Precisely, the load balancer works as follows:
+
+* if the previously selected service instance has not failed, and is still available, return it
+* else if exists, return the first available service instance that has no recorded failure
```

Comment:
```suggestion
* else if the service instance is still available, but has failed during a previous invocation, return the first available service instance that has no recorded failure;
```

Reply:
the corrected sentences is not true. If no service was selected previously, we should fall under this branch too. Also it may happen that the previously selected instance did not fail but is just not availalbe anymore (service discovery-wise)